app/controllers/auth.php

```diff
@@ -452,8 +454,8 @@ function ($projectId, $provider, $code, $state) use ($response, $request, $domai
     ->label('error', __DIR__.'/../views/general/error.phtml')
     ->label('webhook', 'auth.oauth')
     ->label('scope', 'auth')
-    ->label('abuse-limit', 50)
-    ->label('abuse-key', 'ip:{ip}')
+    // ->label('abuse-limit', 100)
```

Comment:
This 2 lines can be uncommented, they should work and the abuse check here actually is important